spec/models/supertype_spec.rb

```diff
@@ -6,8 +6,8 @@
       describe "Supertype #{supertype.id}" do
         it "has the required attributes for #{supertype.id}" do
           expect(supertype.id).to_not be_blank
-          expect(supertype.label).to_not be_blank
-          expect(supertype.description).to_not be_blank
+          expect(I18n.t("supertypes.#{supertype.id}.label")).to_not be_blank
+          expect(I18n.t("supertypes.#{supertype.id}.description")).to_not be_blank
```

Comment:
Just a warning that `t` will silently fail with "translation missing: supertypes.foo.description", so even if the translation doesn't exist, it will never be blank 😬 might be worth changing all of these to `t!`.